gitops/addons/charts/adot-collector/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: opentelemetry
+description: A Helm chart to install otel operator
+type: application
+version: 0.8.0
+appVersion: 0.8.0
```

```suggestion
appVersion: 0.40.0
```
Latest make this match the version of the adot collector image